versions.yaml

```
@@ -298,7 +298,7 @@ externals:
       #   this binary.
       #
       # yamllint disable-line rule:line-length
-      binary: "https://github.com/open-policy-agent/opa/releases/download/v0.55.0/opa_linux_amd64_static"
+      binary: "https://github.com/open-policy-agent/opa/releases/download/v0.55.0/opa_linux_${TARGET_ARCH}_static"
```

```suggestion
      url: "https://github.com/open-policy-agent/opa/releases/download/v0.55.0/"
```

I´d  make this in this way here, and then in the rootfs.sh, I'd do
```
opa_bin_url="$(get_package_version_from_kata_yaml externals.open-policy-agent.meta.binary)/opa_linux_${TARGET_ARCH}_static"
```